internal/cli/cli.go

```diff
@@ -63,6 +63,7 @@ func (c *CLI) Run(ctx context.Context, args ...string) error {
 		cli.Flag("embed", "embed assets").Bool(&cmd.Flag.Embed).Default(false)
 		cli.Flag("hot", "hot reloading").Bool(&cmd.Flag.Hot).Default(true)
 		cli.Flag("minify", "minify assets").Bool(&cmd.Flag.Minify).Default(false)
+		cli.Flag("open", "open browser on dev server startup").Bool(&cmd.Flag.OpenBrowser).Default(false)
```

Comment:
Ah sorry, I might have missed this in the previous review but Flag is meant for global options that need to get passed into the framework. We should treat `OpenBrowser` like Listen below and stick it on the run command.